mne_bids_pipeline/steps/preprocessing/_06b_run_ssp.py

```diff
@@ -103,14 +105,18 @@ def run_ssp(
     rejects = dict(ecg=cfg.ssp_reject_ecg, eog=cfg.ssp_reject_eog)
     avg = dict(ecg=cfg.ecg_proj_from_average, eog=cfg.eog_proj_from_average)
     n_projs = dict(ecg=cfg.n_proj_ecg, eog=cfg.n_proj_eog)
-    ch_name: dict[str, list[str] | None] = dict(ecg=None, eog=None)
+    ch_name: dict[str, str | list[str] | None] = dict(ecg=None, eog=None)
```

Comment:
this should have been the case prior to this PR, as `ch_name["ecg"]` was already getting assigned a string a few lines down. Not sure why `mypy` wasn't complaining before.